src/git.ts

```diff
@@ -276,7 +264,13 @@ export async function isGitFolderBiggerThanMaxSize(): Promise<boolean> {
 
     const repoDirs = _.filter(rootDirContent, isDirectory);
     const sizePromises = _.map(repoDirs, async dir => {
-      const { stdout } = await exec(`cd "${dir}" && git count-objects -v`);
+      let stdout
+      try {
+        stdout = await (await exec("git count-objects -v", { cwd: dir })).stdout;
+      } catch (error) {
+        logger.error("Failed to estimate git repository size", error)
+        return 0;
```

Comment:
I think that if a folder fails here we should just make sure it doesn't exist, or delete it

Reply:
yeah, makes sense